index.js

```diff
@@ -0,0 +1,108 @@
+'use strict';
+
+var benchmark = require('benchmark');
+var micromatch = require('micromatch');
+var Table = require('cli-table2');
+var Z = require('sanctuary-type-classes');
```

Comment:
Nit: I like to sort imports by package name, because `const {foo, bar, baz} = require('quux');` is difficult to sort otherwise. ;)